javascript/webapp/public/examples/companies.json

```diff
@@ -30,7 +30,6 @@
 				{
 					"rel": "input",
 					"profile": "datatable",
-					"name": "companies",
 					"path": "data/products.csv"
```

Comment:
what name does this end up with? "datatable.json"?

Reply:
probably `datatable (3).json`